gapic/src/main/com/google/gapid/perfetto/views/CpuPanel.java

```diff
@@ -138,7 +139,12 @@ private void renderSummary(RenderContext ctx, CpuTrack.Data data, double w, doub
       double mouseX = state.timeToPx(tStart + hovered.bucket * data.bucketSize + data.bucketSize / 2);
       double dx = HOVER_PADDING + hovered.size.w + HOVER_PADDING;
       double dy = HOVER_PADDING + hovered.size.h + HOVER_PADDING;
-      double cardX = Math.min(mouseX + HOVER_MARGIN, w - dx);
+      // By default, the hover card left boundary x-axis coordinate is the current hover point plus
+      // CURSOR_SIZE and HOVER_MARGIN.
+      double cardX = mouseX + CURSOR_SIZE + HOVER_MARGIN;
```

Comment:
Sometimes we have the cursor size counted in but sometimes we don't, if we can be more consistent it would be great.